### PR TITLE
[HDInsight][Hilo]Add new APIs

### PR DESCRIPTION
**Pull request by aim-for-better**
# ARM (Control Plane) API Specification Update Pull Request 

## PR review workflow diagram

Please understand this diagram before proceeding. It explains how to get your PR approved & merged.

diagram

## Purpose of this PR

What's the purpose of this PR? Check all that apply. This is **mandatory**!

  - [ ] New API version. (Such PR should have been generated with [OpenAPI Hub](https://aka.ms/openapihub), per [this wiki doc](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).)
  - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
  - [ ] Update existing version to fix swagger quality issues in S360.
  - [x] Other, please clarify:
   We are adding new RP related APIs under the same resource provider. It is a new API version but this new API version is for new RP instead of for existing RP. We can say we are adding new APIs for new service/RP.

## Due diligence checklist

To merge this PR, you **must** go through the following checklist and confirm you understood 
and followed the instructions by checking all the boxes:

- [x] I have reviewed the general guidance on the spec PR review process: https://aka.ms/specprreview.
- [x] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data-plane related specifications.
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy).
- [ ] I have reviewed following [Resource Provider guidelines](https://aka.ms/rpguidelines), including
  [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can request review from an ARM API Review board.

### ARM API changes review

- If you want for the ARM team to review this PR, you must add the `ARMReview` label. 
- The automation may automatically add the `ARMReview` label, if appropriate.  
  If this happens, proceed according to guidance given in GitHub comments also added by the automation.

### Breaking change review

If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), 
follow the process outlined in the [High-level Breaking Change Process doc](https://aka.ms/breakingchangesprocess#high-level-breaking-change-process).
      
## Getting help

- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- For additional help, see https://aka.ms/azsdk/support/spectools.

### CODE REVIEW

**Comment by aim-for-better** [comment]
We need to suppress Avocado error because we are adding the new RP related APIs, it is by design or expected that the default tag doesn't include the all APIs.

File: specification/hdinsight/resource-manager/readme.go.md [hunk 1]
```diff
@@ -26,6 +26,20 @@ batch:
...
+
+``` yaml $(tag) == 'package-hdinsightonaks-2023-06-preview' && $(go)
+license-header: MICROSOFT_MIT_NO_VERSION
+module-name: sdk/resourcemanager/hdinsight/armhdinsightonaks
+module: github.com/Azure/azure-sdk-for-go/$(module-name)
+output-folder: $(go-sdk-folder)/$(module-name)
+azure-arm: true
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2023-06-01-preview/$(namespace)
+```
+
```
Comment (tadelesh) [inline specification/hdinsight/resource-manager/readme.go.md:42]: This config is for our deprecated version. You shall remove it. Then Go SDK automation failure could be auto-mitigated.

Comment (aim-for-better) [reply specification/hdinsight/resource-manager/readme.go.md]: fixed

**Comment by aim-for-better** [comment]
Hi team, this PR is ready for review. There is only one required validation Swagger Avocado pipeline failed which is expected as we are adding new RP related APIs.
The other failed pipelines like sdk-for-net-track2 or azure-powershell is not mandatory and don't have impact on the APIs. We can skip these sdk/powershell pipelines now.

Please start to review this PR

**Comment by visingla-ms** [comment]
> {

Suggestion: you should name your swagger file as clusterpool.json or clusterpools.json, keeping other existing swagger spec files under this RP.

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>           "ClusterPoolVersionListResult": {

Looks like a type: it should be "ClusterVersionListResult".

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1003 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusterpools/{clusterPoolName}/clusters/{clusterName}/runJob": {

Curious as why this is modelled as Post Action (runJob), and not using PUT (/Microsoft.HDInsight/clusterpools/{clusterPoolName}/clusters/{clusterName}/jobs/{jobName}? That will make it consistent with keeping ClusterJob as ARM Resource.

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:640 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>         "subnetId": {

Looks like this is an ARM Resource, so consider using this: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details).

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1098 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>         "workspaceId": {

Looks like this is an ARM Resource, so consider using this: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details).

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1116 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>         "aksClusterResourceId": {

Looks like this is an ARM Resource, so consider using this: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details).

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1127 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>           "type": "string",

Enum?

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1229 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>         "msiResourceId": {

Looks like this is an ARM Resource, so consider using this: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details).

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1346 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>           "type": "string",

Enum?

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1397 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>         "keyVaultResourceId": {

Looks like this is an ARM Resource, so consider using this: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details).

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1419 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>         "storagekey": {

Is it a secret value? If yes, use this: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret), and make sure your service implemntation doesn't return this in GET/PUT/PATCH API response. Only POST Actions can return secret values, if needed.

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:2283 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by visingla-ms** [comment]
>     "ClusterJob": {

Since you are returning ClusterJob as part of List Response from /jobs Collection GET API, it has to be in ARM Resource format, simple object schema is invalid.

---
Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:2942 in b947edd. [](commit_id = b947eddd576ea2a3b48a81146b742a30e0052596, deletion_comment = False)

**Comment by aim-for-better** [comment]
> > {
> 
> Suggestion: you should name your swagger file as clusterpool.json or clusterpools.json, keeping other existing swagger spec files under this RP.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Hi @visingla-ms , Thanks for your suggestion. I agree with you. But now could you please allow us keep this, change this will cause many efforts like rpaas resource registration. And now we have time buffer limitation. I can create one work item and when we have time we can split it. Thank you very much.

**Comment by aim-for-better** [comment]
> > ```
> >       "ClusterPoolVersionListResult": {
> > ```
> 
> Looks like a type: it should be "ClusterVersionListResult".
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1003 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed. It is an error. Thanks for your comment.

**Comment by aim-for-better** [comment]
> > ```
> > "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusterpools/{clusterPoolName}/clusters/{clusterName}/runJob": {
> > ```
> 
> Curious as why this is modelled as Post Action (runJob), and not using PUT (/Microsoft.HDInsight/clusterpools/{clusterPoolName}/clusters/{clusterName}/jobs/{jobName}? That will make it consistent with keeping ClusterJob as ARM Resource.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:640 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

It is marked as proxy resource. Actually the job related API is the same with this https://github.com/Azure/azure-rest-api-specs-pr/pull/13445 merged after I update it.

**Comment by aim-for-better** [comment]
> > ```
> >     "subnetId": {
> > ```
> 
> Looks like this is an ARM Resource, so consider using this: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1098 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed. Thanks for this comment. I learn it here. The same as the below id related comments.

**Comment by aim-for-better** [comment]
> > ```
> >     "workspaceId": {
> > ```
> 
> Looks like this is an ARM Resource, so consider using this: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1116 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed. Thanks

**Comment by aim-for-better** [comment]
> > ```
> >     "aksClusterResourceId": {
> > ```
> 
> Looks like this is an ARM Resource, so consider using this: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1127 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed. Thanks

**Comment by aim-for-better** [comment]
> > ```
> >     "aksClusterResourceId": {
> > ```
> 
> Looks like this is an ARM Resource, so consider using this: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1127 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed. Thanks.

**Comment by aim-for-better** [comment]
> > ```
> >       "type": "string",
> > ```
> 
> Enum?
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1229 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Hi @visingla-ms  it is readonly and this status will be expanded per business not enum here. Thanks

**Comment by aim-for-better** [comment]
> > ```
> >     "msiResourceId": {
> > ```
> 
> Looks like this is an ARM Resource, so consider using this: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1346 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed. Thanks

**Comment by aim-for-better** [comment]
> > ```
> >       "type": "string",
> > ```
> 
> Enum?
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1397 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed, thanks.

**Comment by aim-for-better** [comment]
> > ```
> >     "keyVaultResourceId": {
> > ```
> 
> Looks like this is an ARM Resource, so consider using this: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:1419 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed, thanks

**Comment by aim-for-better** [comment]
> > ```
> >     "storagekey": {
> > ```
> 
> Is it a secret value? If yes, use this: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret, and make sure your service implemntation doesn't return this in GET/PUT/PATCH API response. Only POST Actions can return secret values, if needed.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:2283 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed, thanks

**Comment by aim-for-better** [comment]
> > ```
> > "ClusterJob": {
> > ```
> 
> Since you are returning ClusterJob as part of List Response from /jobs Collection GET API, it has to be in ARM Resource format, simple object schema is invalid.
> 
> Refers to: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json:2942 in [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596). [](commit_id = [b947edd](https://github.com/Azure/azure-rest-api-specs/commit/b947eddd576ea2a3b48a81146b742a30e0052596), deletion_comment = False)

Fixed, marked it as proxy resoruce. Actually the job related api is the same with the merged PR https://github.com/Azure/azure-rest-api-specs-pr/pull/13445 after my update. Thank  you.

**Comment by aim-for-better** [comment]
Hi @visingla-ms , Thank you very much for your review.
You left 12 comments:
2 for job related(Fixed), 
1 is for file name(hope we can keep the name), 
1 for our error(Fixed), 
2 for change string to enum(Fixed 1and hope we can keep 1), 
1 for x-ms-secret(Fixed)
5 for applying arm-id(Fixed). I resolved 10, and answered 2(hope can keep file name and keep one as string). Thanks.

**Comment by aim-for-better** [comment]
Hi team, this PR is almost the same with the merged private repo PR https://github.com/Azure/azure-rest-api-specs-pr/pull/13385.  
There are only three changes compared the merged one:
1) Thanks for @visingla-ms 's comment, this PR based the merged PR and improve some resource id with arm-id format and add one string to enum and fixed 1 error. 
2) due to public repo validation pipeline is different from private repo, I fixed the validation errors like adding header, update description
3) I reorganized the folder structure so that we can release separate SDKs based this swagger spec.

**Comment by aim-for-better** [comment]
Hi team, if you approved this PR https://github.com/Azure/azure-rest-api-specs/pull/25030, you can also skip this one. They are the same.

The current PR is created by myself manually. The Title"[HubGenerated]" is generated by hub. Both are based the merged private PR and resolved this PR's comments.
Thanks

**Comment by visingla-ms** [comment]
Please fix the Avocado errors.

**Comment by aim-for-better** [comment]
Hi @rkmanda could you please approve the Avocado error due to it is expected and merge this PR?
This is ARM Signed Off status.

**Comment by aim-for-better** [comment]
The avocado failures indicates  [MISSING_APIS_IN_DEFAULT_TAG](https://github.com/Azure/avocado/blob/master/README.md#MISSING_APIS_IN_DEFAULT_TAG), we are adding the new APIs for hilo, and we want to separate the hilo's APIs with the existing HDInsight's APIs. This is the reason that there is Avocado error.

**Comment by aim-for-better** [comment]
this Avocado error need your approval to suppress. We are adding new APIs and adding separate readme.md file.

**Comment by aim-for-better** [comment]
We got ARM signed off per this commit 2773584636477167c98778f2fe33ee7d323e42e3, but after this commit, we still need to update the file.
image

**Comment by konrad-jamrozik** [comment]
Suppressing Avocado failure as per the [discussion on Teams](https://teams.microsoft.com/l/message/19:0351f5f9404446e4b4fd4eaf2c27448d@thread.skype/1690769410062?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1690769410062&teamName=Azure%20SDK&channelName=API%20Spec%20Review&createdTime=1690769410062) and [this comment](https://github.com/Azure/azure-rest-api-specs/pull/24940#issuecomment-1659455096) it appears this is a known limitation of:
- https://github.com/Azure/azure-sdk-tools/issues/6201

File: specification/hdinsight/resource-manager/readme.md [hunk 2]
```diff
@@ -110,6 +110,12 @@ directive:
     reason: Update the default error response to a new format would be a braking change for service.
 ```
 
+```yaml
+directive:
```
Comment (konrad-jamrozik) [inline specification/hdinsight/resource-manager/readme.md:114]: Please remove this code. It doesn't work per [this doc](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process?anchor=obtaining-suppression-for-avocado) and it blocks your PR from being merged.

File: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+
+### Tag: package-2023-06-preview
+These settings apply only when `--tag=package-2023-06-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-06-preview'
+input-file:
+  - preview/2023-06-01-preview/hdinsight.json
+suppressions:
+  - code: MISSING_APIS_IN_DEFAULT_TAG
```
Comment (konrad-jamrozik) [inline specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md:46]: Same remark as [here](https://github.com/Azure/azure-rest-api-specs/pull/24940/files#r1282607706)

Comment (aim-for-better) [reply specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md]: removed, thanks @konrad-jamrozik  for your comment.

File: specification/hdinsight/resource-manager/readme.md (hunk 2 above)
Comment (aim-for-better) [reply specification/hdinsight/resource-manager/readme.md]: removed, thanks @konrad-jamrozik for your help.

File: specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+### Suppression
+
+
+### Tag: package-2023-06-preview
+These settings apply only when `--tag=package-2023-06-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-06-preview'
+input-file:
+  - preview/2023-06-01-preview/hdinsight.json
+suppressions:
```
Comment (konrad-jamrozik) [inline specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md:45]: To get the `Approved-Suppression` label, you need to get this suppressions reviewed and approved.

Comment (aim-for-better) [reply specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md:45]: > To get the `Approved-Suppression` label, you need to get these suppressions reviewed and approved.
> 
> @rkmanda to have a look

Thanks

**Review by rkmanda — APPROVED** [review]

**Pull request merged**